modules/modal/ModalHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { withStyles } from "@diana-ui/base";
+import { ThemeStyleSheetFactory, WithStylesProps } from "@diana-ui/types";
+import { Icon } from "@diana-ui/icon";
+import { SectionTitle, Description } from "@diana-ui/typography";
+
+export interface IProps extends WithStylesProps {
+  title: string;
+  onClose: () => void;
```

Comment:
onClose should be on Modal. And he would pass it to is children (if Modal header). So that modal onclose would be the same for both. You can close via esc, via click overlay, via click close and would be the same